Projekty/Projekt1/Koziel_Krupinski_Lis/kamien_milowy_2.ipynb

```diff
@@ -0,0 +1,2601 @@
+{
```

Comment:
<p>Fajnie że to sprawdziliście, usunięcie jednej z kolumn wydaje się być w tym przypadku dobrym pomysłem. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>